tests/nlu/selectors/test_selectors.py

```diff
@@ -319,7 +319,9 @@ async def test_process_gives_diagnostic_data(trained_response_selector_bot: Path
     # The `attention_weights` key should exist, regardless of there being a transformer
     assert "attention_weights" in diagnostic_data[name]
     # By default, ResponseSelector has `number_of_transformer_layers = 0`
-    assert diagnostic_data[name].get("attention_weights") is None
+    # in which case the attention weights should be an empty numpy array.
```

Comment:
why did it change?

Reply:
In order to be consistent with the rest of the code, `RasaSequenceLayer` now returns `tf.zeros((0,))` (not `None`) for the attention weights in case there's no transformer. That's then changed into an empty numpy array in `models.py/rasa_predict` (by calling `.numpy()` on the tensor). Hence the changed test.

Reply:
hm, let's not propagate empty arrays to other part of codes. Can you convert it to `None` in `models.py/rasa_predict`

Reply:
Sure!